mujoco_warp/_src/sensor_test.py

```diff
@@ -43,22 +43,36 @@ def test_sensor(self):
     """Test sensors."""
     mjm = mujoco.MjModel.from_xml_string("""
       <mujoco>
+        <option gravity="-1 -1 -1"/>
         <worldbody>
-          <body>
+          <body pos="0 0 0">
```

Comment:
nit: pos="0 0 0" is default, no?  should this be deleted or were you trying to document some expectation here?

Reply:
probably wanted to match style among the three bodies with each one having a specified position:
`pos="0 0 0"`, `pos="1 0 0"`, `pos="2 0 0"`

this body `pos` is no longer the default after the merge